sdk/src/main/java/io/opentelemetry/sdk/metrics/aggregator/AggregatorFactory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.metrics.aggregator;
+
+/** Factory class for {@link Aggregator}. */
+public interface AggregatorFactory {
```

Comment:
I'm not sure I understand why this is necessary yet, but I bet you have plans. :)

Reply:
:)